unreleased.sh

```diff
@@ -1,49 +0,0 @@
-#!/usr/bin/env bash
-# Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
-# SPDX-License-Identifier: Apache-2.0
-
-if [ "$#" -ne 1 ]; then
```

Comment:
I'd suggest keeping the change-log markings on a purely optional. This would mean we should be keeping this script for processing of those entries.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/15423/commits/1d5650a110f3bd1ef3b15e45a8d15440b94b8590